README.md

```diff
@@ -1,111 +1,55 @@
+🔎 **Find a [model](https://platform.aixplain.com/discovery/models), [dataset](https://platform.aixplain.com/discovery/datasets), [metric](https://platform.aixplain.com/discovery/metrics) on the platform.**
 
+:yellow_heart: Our repository is constantly evolving. With the help of the scientific community, we plan to add even more datasets, models and metrics across domains and tasks.
 
-## Installation
+## Getting Started
 
-```
+### Installation
+To install simply,
+```bash
 pip install aixtend
 ```
 
-## User Guide
-
-In order to use aiXplain, you need to create an account in [aiXplain platform](https://platform.aixplain.com/). Follow the code samples listed below to get started.
-
-### Code Samples and Demos
-
-aiXtend provides python APIs to call AI workflows you can build with aiXplain designer and to call models you subscribed with aiXplain discover.
-
-#### Pipeline Snippet
-##### Synchronous
-
-```python
-from aixtend.factories.pipeline_factory import PipelineFactory
+###  API Key Setup
+Before you can use the aiXtend SDK, you'll need to obtain an API key from our platform. For details refer this [Team API Key Guide](docs/user/api_setup.md).
 
-api_key = <API_KEY>
-
-pipeline = PipelineFactory.initialize(api_key=api_key)
-
-path = <DATA_URL>
-response = pipeline.run(data=path)
+Once you get the API key, you'll  need to add this API key as an environment variable on your system.
+#### Linux and macOS
+```bash
+export TEAM_API_KEY=YOUR_API_KEY
 ```
-##### Asynchronous
-```python
-import time
-from aixtend.factories.pipeline_factory import PipelineFactory
-
-api_key = <API_KEY>
-
-pipeline = PipelineFactory.initialize(api_key=api_key)
-
-path = <DATA_URL>
-response = pipeline.run_async(data=path)
-if response['status'] != 'FAILED':
-    poll_url = response['url']
-    completed = False
-    while not completed:
-        response = pipeline.poll(poll_url)
-        completed = response['completed']
-        time.sleep(3)
+#### Windows
+```bash
+set TEAM_API_KEY=YOUR_API_KEY
 ```
+## Usage
 
-#### Model Snippet
-##### Synchronous
-```python
-from aixtend.factories.model_factory import ModelFactory
-
-api_key = <API_KEY>
+Let’s see how we can use aiXtend to run a machine translation model.
 
-model = ModelFactory.initialize(api_key=api_key)
 
-path = <DATA_URL>
-response = model.run(data=path)
-```
-##### Asynchronous
 ```python
-import time
-from aixtend.factories.model_factory import ModelFactory
-
-api_key = MODEL_API_KEY
-
-model = ModelFactory.initialize(api_key=api_key)
-
-path = <DATA_URL>
-response = model.run_async(data=path)
-if response['status'] != 'FAILED':
-    poll_url = response['url']
-    completed = False
-    while not completed:
-        response = model.poll(poll_url)
-        completed = response['completed']
-        time.sleep(3)
+from aixtend.factories.model_factory  import ModelFactory
+model = ModelFactory.create_asset_from_id("61b27086c45ecd3c10d0608c") # Got the ID of an MT model from on our platform
```

Comment:
I have 2 comments on this line:
1. Can we create user friendly URIs for referring to models. This can be a combination of supplier, model name and version. It will replace needing ID references in the future.
2. How do we get this ID? We need a simple doc for that. i.e. Change the comment to: Follow the explore section in the user doc to get the ID.

Reply:
1. We can do this. Not sure if search API allows version as a param.  In practice, user would explore pipelines either on the platform or from explore models from the SDK.
2. I do have a doc. I can write the comment as string but can't but an hyperlink in code comment. Let me change that now